integrations/okta/actions/add-group.md

```diff
@@ -6,7 +6,7 @@
 - **Description:** Adds a new group with the OKTA_GROUP type to your org
 - **Version:** 0.0.1
 - **Group:** Others
-- **Scopes:** `okta.groups.manage`
+- **Scopes:** `${PWD}.groups.manage`
```

Comment:
Not sure where that comes from, @khaliqgant?

Reply:
Yeah looks like a bug in the replacement script. Will fix. Thanks!

Reply:
I reverted the change, if you want to merge that as-is